src/routes/api/lynt/+server.ts

```diff
@@ -95,32 +95,13 @@ export const POST: RequestHandler = async ({
 			const buffer = await imageFile.arrayBuffer();
 			const inputBuffer = Buffer.from(buffer);
 
-			const resizedBuffer = await sharp(inputBuffer, {
-				animated: true
-			})
-				.rotate()
-				.webp({ quality: 70 })
-				.withMetadata()
-				.toBuffer();
-
-			const fileName = `${uniqueLyntId}.webp`;
-
-			await minioClient.putObject(
-				process.env.S3_BUCKET_NAME!,
-				fileName,
-				resizedBuffer,
-				resizedBuffer.length,
-				{
-					'Content-Type': 'image/webp'
-				}
-			);
-
+			await uploadCompressed(inputBuffer, uniqueLyntId, minioClient);
```

Comment:
Maybe at some point we can have support for video, audio, and multiple attachments (not just one attachment).